src/bootstrap/bootstrap.py

```diff
@@ -678,7 +678,7 @@ def update_submodule(self, module, checked_out, recorded_submodules):
         run(["git", "submodule", "-q", "sync", module],
             cwd=self.rust_root, verbose=self.verbose)
         run(["git", "submodule", "update",
-            "--init", "--recursive", "--progress", module],
```

Comment:
Why is this change needed?

Reply:
I think this is to work around #57080.  I have an open request in #60379 to try and fix that issue.

Reply:
That is exactly correct, I'm going to revert it on the next commit